examples/ngrok-http2.js

```diff
@@ -0,0 +1,27 @@
+const ngrok = require("../index.js");
+const http2 = require("node:http2");
+
+const server = http2.createServer();
+server.on("error", (err) => console.error(err));
+
+server.on("stream", (stream, headers) => {
+  // stream is a Duplex
+  stream.respond({
+    "content-type": "text/html; charset=utf-8",
```

Comment:
Looks like a small formatting issue here 👀

Reply:
Yeah, its a verbatim copy from Node `http2` docs `https://nodejs.org/api/http2.html#server-side-example` and I feel inclined to let it be.